src/editor/nodes/collapsibleBlock/inputRuleHandlers/chevronHandler.test.ts

```diff
@@ -19,19 +19,19 @@ describe('Chevron input rule handler', () => {
     expect(collapsibleBlocks).toHaveLength(1);
 
     const [collapsibleBlock] = collapsibleBlocks;
-    expect(getText(collapsibleBlock)).toEqual('Some content');
+    expect(getText(collapsibleBlock)).toBe('Some content');
```

Comment:
See https://github.com/veritem/eslint-plugin-vitest/blob/HEAD/docs/rules/prefer-to-be.md